_documentation/user/flags/identify.md

```diff
@@ -5,4 +5,18 @@ type: Good
 summary: Identifies type of nearest flag.
 ---
 
-...
+The Identify (ID) flag allows the player to see into the contents (type) of the nearest flag to the holder.
```

Comment:
I'd specify the fact that the player doesn't have to actually pick up said flag to know what it is.